ci-operator/step-registry/openshift/microshift/infra/iso-build/openshift-microshift-infra-iso-build-commands.sh

```diff
@@ -46,15 +44,12 @@ cp /tmp/pull-secret "\${PULL_SECRET}"
 cd ~/microshift
 
 ./test/bin/ci_phase_iso_build.sh
-
-./scripts/image-builder/build.sh -pull_secret_file "\${PULL_SECRET}" -microshift_rpms ~/rpms -authorized_keys_file ~/.ssh/id_rsa.pub -open_firewall_ports 6443:tcp
```

Comment:
I'm seeing an error on https://github.com/openshift/microshift/pull/1989 that I think will be resolved by removing this step. See https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_microshift/1989/pull-ci-openshift-microshift-main-microshift-metal-tests/1679202926033113088#1:build-log.txt%3A4889 for example

Reply:
This started happening in all builds. The issue is that there is a directory missing, but we didnt change the logic. Not sure if its worth fixing as merging this PR will make it disappear.